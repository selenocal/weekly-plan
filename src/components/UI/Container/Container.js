import React from "react";
import styled from "styled-components";
import PropTypes from "prop-types";

const Container = ({ children }) => <ContainerDiv>{children}</ContainerDiv>;

const ContainerDiv = styled.div`
  max-width: 1100px;
  display: flex;
  flex-direction: column;
  margin: 0 auto;
`;

Container.propTypes = {
  children: PropTypes.node
};

export default Container;
