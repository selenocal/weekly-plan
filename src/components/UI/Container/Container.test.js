import React from "react";
import { shallow } from "enzyme";
import Container from "./Container";

describe("<Container />", () => {
  const wrapper = shallow(<Container>TEST</Container>);

  it("expects render Container correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("expects Container contains given data", () => {
    expect(wrapper.contains("TEST")).toBe(true);
  });
});
