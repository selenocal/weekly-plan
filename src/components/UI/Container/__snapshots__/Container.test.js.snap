// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> expects render Container correctly 1`] = `ShallowWrapper {}`;
